TAScheduler/tests/AcceptanceTests/test_CreateUser.py

```diff
@@ -0,0 +1,166 @@
+                              "password": "supervisorTest",
+                              "name": "Test Supervisor", "address": "address",
+                              "phone": "123456789123",
+                              "email": "s@s.com"}, follow=True)
+
+        self.assertEqual(r.context["error"], "User was not created. A phone number needs to be 10 digits",
+                         "An error message was not displayed when password was left blank")
+        self.assertEqual(UserProfile.objects.count(), 1, "Database did not change")
+
+    def test_blankEmail(self):
+        r = self.client.post("/create_user/",
+                             {"userID": 2, "userType": "SUPERVISOR", "username": "supervisorTest",
+                              "password": "supervisorTest",
+                              "name": "Test Supervisor", "address": "address",
+                              "phone": 1234567891,
+                              "email": ""}, follow=True)
+
+        self.assertEqual(r.context["error"], "User was not created. Email should not be left blank",
+                         "An error message was not displayed when password was left blank")
+        self.assertEqual(UserProfile.objects.count(), 1, "Database did not change")
+
+    def test_createSupervisor(self):
+        r = self.client.post("/create_user/",
+                             {"userID": 98, "userType": "SUPERVISOR", "username": "supervisorTest",
+                              "password": "supervisorTest",
+                              "name": "Test Supervisor", "address": "address",
+                              "phone": 1234567891,
+                              "email": "s@s.com"}, follow=True)
+
+        self.assertEqual(UserProfile.objects.get(userID=98).userType, "SUPERVISOR",
+                         "Supervisor was created successfully")
+
+    def test_createInstructor(self):
+        r = self.client.post("/create_user/",
+                             {"userID": 99, "userType": "INSTRUCTOR", "username": "InstructorTest",
+                              "password": "InstructorTest",
+                              "name": "Test Instructor", "address": "address",
+                              "phone": 1234567891,
+                              "email": "i@i.com"}, follow=True)
+
+        self.assertEqual(UserProfile.objects.get(userID=99).userType, "INSTRUCTOR",
+                         "Instructor was created successfully")
+
+    def test_createTA(self):
+        r = self.client.post("/create_user/",
+                             {"userID": 100, "userType": "TA", "username": "taTest",
+                              "password": "taTest",
+                              "name": "Test Ta", "address": "address",
+                              "phone": 1234567891,
+                              "email": "t@t.com"}, follow=True)
+
+        self.assertEqual(UserProfile.objects.get(userID=100).userType, "TA", "TA was created successfully")
```

Comment:
```suggestion
        self.assertEqual(UserProfile.objects.get(userID=100), UserProfile(userID=100, userType="TA", username="taTest", password="taTest", name="Test Ta", address="address", phone=1234567891, email="t@t.com"), "TA was not created successfully")
```
The message displays if the test fails, so it should tell the user what happened if the test failed. Also, if you're testing if the user was created successfully, it's more complete to test it against a user with all the fields the same instead of just userType.